components/hab/src/command/studio.rs

```diff
@@ -197,13 +197,13 @@ mod inner {
 
         // All the args already placed in `cmd_args` are things that we don't want to insert again.
         // Later args such as `--env` will overwrite any options (potentially) set mistakenly here.
-        if let Ok(ops) = henv::var(DOCKER_OPS) {
+        if let Ok(ops) = henv::var(DOCKER_OPTS) {
```

Comment:
@eeyun since we are in here, why not change the ops to opts for the local variable name?